setup.py

```diff
@@ -17,17 +17,17 @@
     packages=find_packages(exclude=['sandbox*', 'tests*']),
     include_package_data=True,
     install_requires=[
-        'django>=1.11,<2.2',
+        'django>=1.11,<2.2.8',
         'requests>=1.0',
         'django-localflavor'
     ],
     extras_require={
-        'oscar': ['django-oscar>=1.6,<1.7']
+        'oscar': ['django-oscar>=2.0']
```

Comment:
Please pin to `<2.1` to avoid breakages on the next release of Oscar until it's tested.